op-bindings/predeploys/addresses_test.go

```diff
@@ -36,5 +36,5 @@ func TestL1BlockSlots(t *testing.T) {
 
 	require.Equal(t, types.OverheadSlot, overHeadSlot)
 	require.Equal(t, types.ScalarSlot, scalarSlot)
-	require.Equal(t, types.L1BaseFeeSlot, l1BaseFeeSlot)
+	require.Equal(t, types.L1BasefeeSlot, l1BaseFeeSlot)
```

Comment:
if we are reverting the `Basefee` -> `basefee`, this should be included

Reply:
this constant comes from op-geth, where I'll need to revert the change separately.